src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts

```diff
@@ -109,6 +117,12 @@ export class EventStoreComponent implements OnInit, OnDestroy {
 
   public simulEvents: IConcurrentEvents[];
 
+  public magicWand: { [K: string]: Function } = {
```

Comment:
move this to the transform.ts, for now we can just have this list initialized in the file. While it would be beneficial to later make it extendable, this isnt something we will do in SFX.